src/modules/transaction/constants.js

```diff
@@ -0,0 +1,7 @@
+export const FEE_TYPES = {
+  BYTES_FEE: 'bytesFee',
+  PRIORITY_FEE: 'priorityFee',
+  INITIALIZE: 'initialize',
+  INITIALIZE_ESCROW: 'initializeEscrow',
+  REGISTER_VALIDATOR: 'registerValidator',
```

Comment:
```suggestion
  USER_ACCOUNT_INITIALIZATION: 'userAccountInitialization',
  ESCROW_ACCOUNT_INITIALIZATION: 'escrowAccountInitialization',
  VALIDATOR_REGISTRATION: 'validatorRegistration',
```
lets try to express as much closely possible to protocol https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/token/endpoint.ts#L184

https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/pos/endpoint.ts#L335